SimpleRss/SimpleRss/Views/FeedViewCell.swift

```diff
@@ -13,8 +13,17 @@ class FeedViewCell: UITableViewCell {
     @IBOutlet weak var previewImageView: UIImageView!
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var pubDateLabel: UILabel!
     
     override func prepareForReuse() {
         previewImageView.image = nil
+        descriptionLabel.lineBreakMode = .byTruncatingTail
+        descriptionLabel.numberOfLines = 1
     }
+    
+    func expanding(isExpanded: Bool) {
```

Comment:
Wouldn't it be better to make this a property and put this logic in a setter?

Reply:
Do you mean this approach?
```suggestion
   var isExpanded: Bool {
        get { return expanded }
        set { expanded = newValue }
    }
    
    let feed = feedList[indexPath.row]
    feed.isExpanded != feed.isExpanded
```

Reply:
```suggestion
    var isExpanded: Bool {
        didSet {
            descriptionLabel.numberOfLines = isExpanded ? 0 : 1;
            descriptionLabel.lineBreakMode = isExpanded ? .byCharWrapping : .byTruncatingTail
        }
    }
```